tests/integration/test_issues.py

```diff
@@ -130,3 +137,42 @@ def test_issue_1304_arc_sketch_creation():
     finally:
         # Reverse the default units to meter
         DEFAULT_UNITS.LENGTH = UNITS.meter
+
+
+def test_issue_1309_revolve_operation_with_coincident_origins(modeler: Modeler):
```

Comment:
I'm not fond of having the issue reference in the name of the test, it doesn't add info to what the test is testing and we already have the reference in the description. Isn't `revolve_operation_with_coincident_origins` enough to describe the test?

Reply:
The tests located in ``test_issues`` are tests coming from the issue descriptions provided by users. We are verifying that the code provided by them now works and continues to work in the future. Although the issue link is available on the docstring, this allows to quickly identify failing tests and their related issues without having to navigate internally.

This is a specific module only for testing code snippets coming from bug issues.

And ``revolve_operation_with_coincident_origins``  is a generic test I came up with.. I still want to test the use case provided by users - even if it implies testing some functionalities several times